lib/proto/attach-form.js

```diff
@@ -31,6 +31,22 @@ var formAction = function(el, event) {
 
   event.preventDefault()
 
+  if (el.enctype === "multipart/form-data") {
+    options.requestOptions.formData = new FormData(el)
+  }
+  else {
+    options.requestOptions.requestParams = parseFormElements(el)
```

Comment:
You can remove `requestParams: []` from line 19 now, as this makes that initialisation redundant.

Reply:
Fixed.